src/main/drivers/persistent.h

```diff
@@ -29,10 +29,18 @@ typedef enum {
     PERSISTENT_OBJECT_MAGIC = 0,
     PERSISTENT_OBJECT_HSE_VALUE,
     PERSISTENT_OBJECT_OVERCLOCK_LEVEL,
+#ifdef STM32H7
+    // SPRACING H7 firmware always leaves this value reset
```

Comment:
As per my comment on persistentObjectWrite() above. Should I abstract this functionality so that we can make it target specific?

Reply:
I think the least that should be done is to make this workaround conditional to `USE_BROKEN_BOOTLOADER_WORKAROUND` or similar so that non-affected targets (including H7) can be built without this.

Reply:
Can I put this in this PR, or raise another one for that? Or perhaps I should add it here, we test, and then split it out once we're happy?

Reply:
Leave it in here for now - since at the moment SPRacing H7s is the only thing available to you for testing.

Reply:
Done